src/test/scala/no/ndla/articleapi/UnitSuite.scala

```diff
@@ -22,9 +22,12 @@ abstract class UnitSuite extends FunSuite with Matchers with OptionValues with I
   setEnv("SEARCH_REGION", "some-region")
   setEnv("RUN_WITH_SIGNED_SEARCH_REQUESTS", "false")
 
-  setEnv("MIGRATION_HOST", "some-host")
-  setEnv("MIGRATION_USER", "some-user")
-  setEnv("MIGRATION_PASSWORD", "some-password")
+  setEnv("AUDIO_API_URL", "localhost:30014")
```

Comment:
Dette betyr at du må ha disse to API'ene kjørende i din lokale Docker?
Hvordan feiler det hvis du ikke har det? Får du en fornuftig melding?